static/app/components/events/contextSummary/utils.tsx

```diff
@@ -21,6 +21,16 @@ export const makeContextFilter = (event: Event) =>
       }
     }
 
+    if (event.sdk?.name === 'sentry.javascript.nextjs') {
```

Comment:
Would be nice if we had a util function that was like `isServerSideRenderedEvent` to encompass other things like remix and gatsby. Although I'm not sure if those SDKs work similarly. Just a though.

Reply:
that's a good idea, i think for now , we want to just test it on NextJS, but if we decide to move this to other frameworks, i agree, this util function could be good.